app/views/hyrax/base/_relationships_parent_rows.html.erb

```diff
@@ -0,0 +1,32 @@
+<%# Render presenters which aren't specified in the 'presenter_types' %>
+<% presenter.grouped_presenters(except: presenter.presenter_types).each_pair do |model_name, items| %>
+  <%= render 'relationships_parent_row', type: model_name, items: items, presenter: presenter %>
+<% end %>
+
+<dt> <%= "Parents" %> </dt>
```

Comment:
I'll extract this into its own partial to keep the code isolated. But I don't expect it to change shape much. Trying to keep this as basic as possible.